README.md

```diff
@@ -1,330 +1,307 @@
-# ROBLOX IS KILLING LOCAL CLIENTSETTINGS LOADING
-# NO MORE CLOSET CHEATERS
-![drake](https://github.com/pizzaboxer/bloxstrap/assets/67791946/303dfde5-1968-435e-8660-249e8b6001c6)
+> [!NOTE]
+> Local Fast Flags are being patched out. Expect your flags to not work soon enough.
```

Comment:
This should be "Local ClientSettings loading is being removed. Expect FastFlags to not be modifiable by the client soon."

Reply:
The word FastFlags are more known than ClientSettings. I will change it a little bit though.